src/components/standup/FormInputComponent.tsx

```diff
@@ -20,7 +19,7 @@ const FormInputComponent: FC<InputProps> = ({
             </label>
             <input
                 id={inputId}
-                data-testid={dataTestId}
+                role="textbox"
```

Comment:
do we need to add this role, I think roles should be accessible through the accessibility role when we call the getByRole API.

Reply:
fixed